nixos/modules/hardware/opentabletdriver.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.hardware.opentabletdriver;
+in
+{
+  options = {
+    hardware.opentabletdriver = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = ''
+          Enable OpenTabletDriver udev rules and blacklist kernel modules known
+          to conflict with OpenTabletDriver.
```

Comment:
This also starts the service. Maybe just say "enable"? Although the blacklist warning does seem interesting.

Reply:
@kevincox Fixed, WDYT?